app/views/organizations/fosterer_invitations/_form.html.erb

```diff
@@ -0,0 +1,32 @@
+<!-- form -->
+<%= bootstrap_form_with model: user, url: invitation_path(user) do |form| %>
```

Comment:
This form is called from the `FosterersInvitationsController` but it is posting to the `invitation_path(user)` which directs to `InvitationsController`, where devise_invitable is setup.